cf-usb-release/packages/cf-usb/pre_packaging

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
 set -e -x
 
+mkdir "${BUILD_DIR}/cf-usb/"
+
 pushd $RELEASE_DIR/..
-source ./make/include/versioning.sh
+    source ./make/include/versioning.sh
+    cp -r * "${BUILD_DIR}/cf-usb/"
```

Comment:
Equivalent comment to https://github.com/SUSE/groot-btrfs/pull/9/files#r235832942